packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -105,7 +105,7 @@ export class RouterliciousRestWrapper extends RestWrapper {
         );
     }
 
-    private generateHeaders(requestHeaders?: AxiosRequestHeaders | undefined): Record<string, string> {
+    private generateHeaders(requestHeaders?: AxiosRequestHeaders | undefined): Record<string, any> {
```

Comment:
@alexvy86 I was running into a build issue with the error: "AxiosHeaders is not assignable to type string". Is this workaround ok to address the issue?

Reply:
Let's do [what we did in main](https://github.com/microsoft/FluidFramework/pull/18848/files#diff-1e2e32a43c0c2dcf17cac109f141eac9278f92537f108f74f52b0795cd418bbb) instead.

Reply:
I tried importing `RawAxiosRequestHeaders` but it doesn't seem to be available in axios 0.x versions. Should we upgrade axios to 1.x instead of 0.28.0?

Reply:
Oh, I see... no, then what you did is fine. Let's keep the axios update as safe as possible, bumping all the way to 1.6 seems a bit riskier than just 0.28.